sites/partners/tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
   "extends": "../../tsconfig.json",
   "compilerOptions": {
+    "target": "es2017",
```

Comment:
This was needed to use the `Set` functionality.

Reply:
We might want to spike on frontend compiler targets generally speaking, as I'm not sure if we're currently shipping a bunch of polyfilles or other questionable aspects of how JS code gets compiled based on browser versions we wouldn't need to support. (e.g., even es2017 is quite old by now, we might want to aim for at least es2020)

Reply:
That's a good call out, I just chose 2017 because it is referenced in shared-helpers in the hopes that it would reduce the number of targets in our system. However, it doesn't make a big difference since we are already using several different targets. I'll bump it up to es2020 for this PR and make sure the build still works as expected